client/src/lib/firebase/firestore-functions-types.ts

```diff
@@ -0,0 +1,4 @@
+import type { GameState } from "./firestore-types/lobby";
+
+export type ChangeState = { code: string; state: GameState };
+export type ErrorResponse = { error?: string };
```

Comment:
we should probably share types between the client and the server like we did for firestore-types